New package: StippleTabs v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: StippleTabs
- Repository: https://github.com/GenieFramework/StippleTabs.jl
- Created by: @essenciary
- Version: v1.0.0
- Commit: bd4e4f75a16fa0aa10453766b43754d5bf927143
- Reviewed by: @essenciary
- Reference: https://github.com/GenieFramework/StippleTabs.jl/commit/bd4e4f75a16fa0aa10453766b43754d5bf927143#commitcomment-146274926
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd61bce217047ff1a9bcf8712100df75582791a8498b5fe25fb077a750b9058e462456d721536adc1a9d6e4e9070abc956a23d923d9d4cf33a555232ba71eb2be49ba8c2311d084f82004a2d1b1bdc842cc3d3f3caa6cf10e60e26d1d6f347e59d499a51228622e0dfd86c9853d9efce0108ba7d65ad3e94d120905a653e5d864db0e3a8b379cd0c9bea25a5e733d152d73ff2e6d344cad0525a1863f0865b1dbe7c535b5d7ee903bf990317d128788707d -->

goerz commented:
Could you extend the README a little bit to give some context and show how the package might be used?

JuliaRegistrator committed:
New package: StippleTabs v1.0.0

UUID: 29f11006-3202-4091-a17b-0a26282d76dd
Repo: https://github.com/GenieFramework/StippleTabs.jl.git
Tree: f8721b7491bfd239c69eace4c4acb0e74962c28f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

essenciary commented:
[noblock]
